fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/StaffTest.java

```diff
@@ -91,6 +92,8 @@ public void testStaffCreateValidationError() {
         StaffHelper.createStaffWithJson(requestSpec, responseSpecForValidationError, new Gson().toJson(map));
     }
 
+    @Test
+    @Ignore //TODO FINERACT-1005
```

Comment:
Create an issue for this ingored test so someone should fix it in the near future.

Reply:
I have created an issue for it.